mypyc/ir/rtypes.py

```diff
@@ -234,9 +234,14 @@ def __repr__(self) -> str:
 short_int_rprimitive = RPrimitive('short_int', is_unboxed=True, is_refcounted=False,
                                   ctype='CPyTagged')  # type: Final
 
-# low level integer (corresponds to C's 'int').
-c_int_rprimitive = RPrimitive('c_int', is_unboxed=True, is_refcounted=False,
+# low level integer (corresponds to C's 'int's).
+c_int32_rprimitive = RPrimitive('c_int32', is_unboxed=True, is_refcounted=False,
```

Comment:
I have a few nits.

I think that we can leave the `c_` prefix out, since "32-bit integer" is a fairly generic concept not specific to C and not likely to be confused with Python types.

Also, the correspondence to C `int` is a bit of an accident (and not generally true, on all possible platforms). It would be better to use `int32_t`, which happens to be the same as `int` on all supported platforms.